prow/cmd/deck/static/prow/prow.ts

```diff
@@ -170,6 +170,24 @@ function handleUpKey(): void {
     adjustScroll(previousSibling);
 }
 
+/**
+ * Gets cookie by its name.
+ */
+function getCookieByName(name: string): string {
```

Comment:
We [already have a `getCookieByName`](https://github.com/kubernetes/test-infra/blob/6d003b6b14ae7c33513fa656a37af3d1c404aff3/prow/cmd/deck/static/pr/pr.ts#L185-L198). If possible, it would be preferable if you could pull these out to a shared library.

Reply:
Would this mean putting it in [common.ts](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/deck/static/common/common.ts)?

Reply:
That seems reasonable enough (or make common/cookies.ts or similar and put it there).